cmd/calyptia/create_pipeline.go

```diff
@@ -149,6 +156,7 @@ func newCmdCreatePipeline(config *config) *cobra.Command {
 	fs.StringVar(&secretsFormat, "secrets-format", "auto", "Secrets file format. Allowed: auto, env, json, yaml. Auto tries to detect it from file extension")
 	fs.StringArrayVar(&files, "file", nil, "Optional file. You can reference this file contents from your config like so:\n{{ files.myfile }}\nPass as many as you want; bear in mind the file name can only contain alphanumeric characters.")
 	fs.BoolVar(&encryptFiles, "encrypt-files", false, "Encrypt file contents")
+	fs.StringVar(&image, "image", "", "Fluent-bit docker image")
```

Comment:
Do we want this to be more specific, e.g. `pipeline-image`? Just thinking if the user will not be sure which image it is particularly if we have more in the future?

Reply:
Its a little bit redundant since we are already in the context of creating a pipeline.

```bash
calyptia create pipeline --pipeline-image HERE
```

🤔

Reply:
Ah I see, yeah no worries.